diagrams/__init__.py

```diff
@@ -414,8 +414,12 @@ def _rand_id():
         return uuid.uuid4().hex
 
     def _load_icon(self):
-        basedir = Path(os.path.abspath(os.path.dirname(__file__)))
-        return os.path.join(basedir.parent, self._icon_dir, self._icon)
+        if self._icon_dir is None or not self._icon_dir.startswith(os.path.sep):
```

Comment:
There is [`os.path.isabs` function](https://docs.python.org/3.6/library/os.path.html#os.path.isabs), how about use it?